HousingSearchApi/V1/Gateways/Models/QueryableIdentification.cs

```diff
@@ -0,0 +1,19 @@
+using Nest;
+
+namespace HousingSearchApi.V1.Gateways.Models
```

Comment:
Hi Anna, why do we need this class? I know it's in the domain but we actually don't have the data migrated (and no plans to migrate it for now either)

Reply:
As I can see now, @evak6979 added almost same class 4 days ago, So I just removed this one from my PR
https://github.com/LBHackney-IT/housing-search-api/blob/master/HousingSearchApi/V1/Gateways/Models/Persons/QueryableIdentification.cs